main.py

```diff
@@ -35,16 +39,17 @@ def load_from_file_():
 TEXT_FIELD_HEIGHT = 21
 LABEL_HEIGHT = 23
 
-MOCK_DATA = {1: ["John", "IT", "Senior Developer", "1500", "40"],
-             5: ["Dave", "Logistics", "Manager", "3000", "35"],
-             3: ["Max", "PR", "Team leader", "2000", "40"]}
+MOCK_DATA = {1: ["John", "IT", "Senior Developer", 1500.0, 40],
+             5: ["Dave", "Logistics", "Manager", 3000.0, 35],
+             3: ["Max", "PR", "Team leader", 2000.0, 40]}
```

Comment:
IMO the MOCK_DATA should be parametrized and NOT available in production.
I'm leaving u the decision, it could be fixed as part of the separate issue, then please create it and leave it in the backlog bucket please. :)

Reply:
Done